Highway.h

```diff
@@ -2,15 +2,20 @@
 
 #include <vector>
 #include <algorithm>
+#include "Vehicle.h"
+
+struct HighwayPatrol;
 
 struct Highway
 {
     void changeSpeed(int newSpeed);
     void addVehicle(Vehicle* v);
     void removeVehicle(Vehicle* v);
+    std::vector<Vehicle*> vehicles;
+    int speedLimit = 65;
```

Comment:
These were private for a reason.
they need to stay private.
Figure out how to give HighwayPatrol access without changing their access specifier
- [ ] revise